### DISCUSSION TITLE
Can Bleak make use of a dongle with an embedded chip?

### DISCUSSION THREAD

**Discussion by Jonathan-Balat** [hbldh/bleak#514] [Q&A]
We have a Windows computer that uses a BLE Dongle that uses an NRF52840 chip to connect to a BLE-ready MCU, but apparently does not appear after scanning for devices. While using PC/phone confirms that the MCU is discoverable/connectable.

**Comment by koenvervloesem** [comment]
I actually did this recently on my Ubuntu machine, with an nRF52840 dongle which I flashed with Zephyr's [hci_usb sample code](https://github.com/zephyrproject-rtos/zephyr/tree/master/samples/bluetooth/hci_usb). I had to specify the device like this:

```
devices = await BleakScanner.discover(device="hci1")
```

Where `hci1` is the device name for the dongle on my machine. After that, Bleak discovered all my BLE devices using the BLE dongle.

I haven't tested this on Windows, as I don't have a Windows system.
